app/views/catalog/_find_this_item.html.erb

```diff
@@ -1 +1 @@
-<%= render 'metadata_block', document: document, presenter: FindThisItemPresenter, presenter_class: 'find-this-item', title: 'Find This Item', add_class_dt: "col-md-12", add_class_dd: "col-md-12" %>
```

Comment:
Is this `metadata_block` partial used elsewhere?

Reply:
Yes.